cypress/component/reference/MultipleEntryReferenceEditor.spec.tsx

```diff
@@ -130,4 +136,292 @@ describe('Multiple Reference Editor', () => {
     findLinkExistingBtn().click(); // inserts 1 card
     findLinkExistingBtn().should('not.exist'); // limit reached, button hidden.
   });
+
+  it(`shows status of entries`, () => {
+    const sdk = createReferenceEditorTestSdk({
+      initialValue: [
+        asLink(fixtures.entry.empty),
+        asLink(fixtures.entry.changed),
+        asLink(fixtures.entry.published),
+      ],
+    });
+    mount(<MultipleEntryReferenceEditor {...commonProps} sdk={sdk} />);
+
+    cy.findAllByTestId('cf-ui-entry-card').should('have.length', 3);
+    cy.findAllByTestId('cf-ui-entry-card').eq(0).findByText('draft').should('be.visible');
+    cy.findAllByTestId('cf-ui-entry-card').eq(1).findByText('changed').should('be.visible');
+    cy.findAllByTestId('cf-ui-entry-card').eq(2).findByText('published').should('be.visible');
+  });
+
+  //Do we also want to test that the props are correct, or should that be done in a jest test?
+  it(`provides the custom card render method with necessary properties`, () => {
+    const sdk = createReferenceEditorTestSdk({
+      initialValue: [asLink(fixtures.entry.published)],
+    });
+    mount(
+      <MultipleEntryReferenceEditor
+        {...commonProps}
+        renderCustomCard={() => {
+          return <Note testId="custom-card" title="Custom card" />;
+        }}
+        sdk={sdk}
+      />
+    );
+
+    cy.findByTestId('custom-card').should('be.visible');
+  });
+
+  it('shows localized displayField as title using the current locale', () => {
+    const sdk = createReferenceEditorTestSdk({
+      initialValue: [asLink(fixtures.entry.published)],
+    });
+    sdk.field.locale = 'de-DE';
+    mount(<MultipleEntryReferenceEditor {...commonProps} sdk={sdk} />);
+
+    findDefaultCards()
+      .eq(0)
+      .findByTestId('title')
+      .should('have.text', 'Der beste Artikel aller Zeiten');
+  });
+
+  it('shows unlocalized displayField as title using the default locale', () => {
+    const sdk = createReferenceEditorTestSdk({
+      initialValue: [asLink(fixtures.entry.published)],
+    });
+    mount(<MultipleEntryReferenceEditor {...commonProps} sdk={sdk} />);
+
+    findDefaultCards().eq(0).findByTestId('title').should('have.text', 'The best article ever');
+  });
+
+  it('shows status of entries', () => {
```

Comment:
This looks like a duplicate of the first test

Reply:
It absolutely is, thanks